src/contracts/storage/position.rs

```diff
@@ -193,16 +198,18 @@ impl Position {
             last_block_number: block_number,
             tokens_owed_x: TokenAmount::new(0),
             tokens_owed_y: TokenAmount::new(0),
-            created_at: current_timestamp,
+            created_at: current_timestamp_in_milliseconds,
+            seconds_per_liquidity_inside: SecondsPerLiquidity::new(0),
         };
 
+        let current_timestamp_in_seconds = current_timestamp_in_milliseconds / 1000;
         let (required_x, required_y) = unwrap!(position.modify(
             pool,
             upper_tick,
             lower_tick,
             liquidity_delta,
             true,
-            current_timestamp,
+            current_timestamp_in_seconds,
```

Comment:
There is no place where we need a timestamp in milliseconds, so it’s worth modifying the function that retrieves the timestamp to return it in seconds to avoid conversions and potential errors.
`get_timestamp`